apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-palette/dot-palette.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, Input } from '@angular/core';
+import { DotCMSContentType } from '@dotcms/dotcms-models';
+
+@Component({
+    selector: 'dot-palette',
+    templateUrl: './dot-palette.component.html',
+    styleUrls: ['./dot-palette.component.scss']
+})
+export class DotPaletteComponent {
+    @Input() items: DotCMSContentType[] = [];
+    contentTypeVariable: string = '';
+
+    constructor() {}
+
+    switchView(variableName?: string): void {
+        this.contentTypeVariable = variableName ? variableName : '';
```

Comment:
If it's only 2 state, shouldn't this be a boolean?

Reply:
the `contentTypeVariable` value is also passed to the `dot-palette-contentlets` component 
![image](https://user-images.githubusercontent.com/37185433/141830191-3da3afa4-c0c9-401b-ba55-411e65b11985.png)